### PR TITLE
add Zero-K

### PR DESCRIPTION
**Pull request by mzemb**

### CODE REVIEW

File: games/z.yaml [hunk 1]
```diff
@@ -103,6 +103,24 @@
   video:
     youtube: RKmKpTe-gVo
 
+- name: Zero-K
+  type: remake
```
Comment (cxong) [inline games/z.yaml]:
```suggestion
  type: similar
```

File: games/z.yaml [hunk 2]
```diff
@@ -103,6 +103,24 @@
...
+  lang:
+  - C#
+  license:
+  - GPL
```
Comment (cxong) [inline games/z.yaml]:
```suggestion
  - GPL2
```

File: games/z.yaml [hunk 3]
```diff
@@ -103,6 +103,24 @@
   video:
     youtube: RKmKpTe-gVo
 
+- name: Zero-K
+  type: remake
+  lang:
+  - C#
```
Comment (cxong) [inline games/z.yaml]:
```suggestion
  - C#
  - Lua
```

File: games/z.yaml [hunk 4]
```diff
@@ -103,6 +103,24 @@
...
+  license:
+  - GPL
+  development: active
+  originals:
+  - Total Annihilation
+  - Supreme Commander
+  status: playable
+  repo: https://github.com/ZeroK-RTS
+  url:
```
Comment (cxong) [inline games/z.yaml]: url only supports one entry

File: games/z.yaml [hunk 5]
```diff
@@ -103,6 +103,24 @@
...
+  originals:
+  - Total Annihilation
+  - Supreme Commander
+  status: playable
+  repo: https://github.com/ZeroK-RTS
+  url: 
+   - https://zero-k.info/
+   - https://store.steampowered.com/app/334920/ZeroK/
+  video:
+   youtube: dyhzqcf7NJM
```
Comment (cxong) [inline games/z.yaml]:
```suggestion
    youtube: dyhzqcf7NJM
  updated: 2019-10-21
```

File: games/z.yaml (hunk 4 above)
Comment (mzemb) [reply games/z.yaml]: I think both entries are equally important here. :/

**Pull request closed without merging**